packages/nvidia_nat_memmachine/tests/test_memmachine_integration.py

```
@@ -0,0 +1,431 @@
+        
+        # Add the memory
+        await memory_client.add_items([direct_memory])
+        
+        # Wait for memory ingestion
+        # Memories are processed asynchronously by MemMachine's background task
+        import asyncio
+        await asyncio.sleep(5)  # Wait for background ingestion task
+        
+        # Try searching multiple times with retries (memory ingestion is async)
+        retrieved_memories = []
+        for attempt in range(3):
```

Ruff (0.14.13)

Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)